css/components/app/pages/explore.scss

```diff
@@ -31,7 +31,41 @@
   }
 
   .filters-sum-up {
-    color: $dark-pink;
+    display: flex;
```

Comment:
the same as the other comments. It should be a component, explore.scss is not a file where you can add whatever you want